tests/components/rainforest_raven/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,107 @@
+# serializer version: 1
+# name: test_entry_diagnostics
+  dict({
+    'config_entry': dict({
+      'data': dict({
+        'device': '/dev/ttyACM0',
+        'mac': '**REDACTED**',
+      }),
+      'disabled_by': None,
+      'discovery_keys': dict({
+      }),
+      'domain': 'rainforest_raven',
+      'entry_id': '01JADXBJSPYEBAFPKGXDJWZBQ8',
+      'minor_version': 1,
+      'options': dict({
+      }),
+      'pref_disable_new_entities': False,
+      'pref_disable_polling': False,
+      'source': 'user',
+      'title': 'Mock Title',
+      'unique_id': None,
+      'version': 1,
+    }),
+    'data': dict({
+      'Meters': dict({
+        '**REDACTED0**': dict({
+          'CurrentSummationDelivered': dict({
+            'device_mac_id': '**REDACTED**',
+            'meter_mac_id': '**REDACTED**',
+            'summation_delivered': '23456.7890',
+            'summation_received': '00000.0000',
+            'time_stamp': None,
+          }),
+          'InstantaneousDemand': dict({
+            'demand': '1.2345',
+            'device_mac_id': '**REDACTED**',
+            'meter_mac_id': '**REDACTED**',
+            'time_stamp': None,
+          }),
+          'PriceCluster': dict({
+            'currency': dict({
+              '__type': "<enum 'Currency'>",
+              'repr': "<Currency.USD: 'USD'>",
```

Comment:
Small note: the diagnostic was failing on my machine because the version `iso4217` was older on my machine and newer on the CI.
See https://github.com/dahlia/iso4217/commit/8cb9c58b7c0ed7efcdcc726411043e5d2e669988